### ISSUE TITLE
Goldpinger-5.3.0 and 5.4.0 is broken

### ISSUE THREAD

**Issue by evgkrsk** [okgolove/helm-charts#67]
**Is this a request for help?**: yes

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): bug report

**Version of Helm and Kubernetes**: any versions

**Which chart**: goldpinger-5.4.0

**What happened**:

```
  Error: Failed to render chart: exit status 1: Error: YAML parse error on goldpinger/templates/daemonset.yaml: error converting YAML to JSON: yaml: line 54: did not find expected key
```

**What you expected to happen**: successful chart upgrade

**How to reproduce it** (as minimally and precisely as possible): write values.yaml with "resources:" section (cpu+mem, requests+limits) and try to install chart with this values file.

**Anything else we need to know**: seems like there is unneeded spaces in templates/daemonset.yaml just before reference to .Values.resources

**Comment by okgolove** [comment]
@evgkrsk hello! Thank you for notifying.
Fixed in #68

**Comment by tjanson** [comment]
Hi @okgolove, thanks for the chart. The same issue applies to the `.Values.extraEnv` include:
https://github.com/okgolove/helm-charts/blob/6c857a10e5834f65ea84444a0d9b1887df7b095f/charts/goldpinger/templates/daemonset.yaml#L51

Would you like a separate issue?
